docs/pages/inboxes/build-inbox.md

```diff
@@ -0,0 +1,160 @@
+# Build the inbox
+
+## Check if an address is reachable on V3
```

Comment:
It still feels like we are treating v3 as "a version of XMTP" and not just "XMTP". V3 is not a special consideration in the post-V2 world!